packages/playwright/src/generators/visualRegression/utils/update-tasks-yamls.ts

```diff
@@ -51,21 +57,8 @@ export function updateTaskctlYaml(
     }
 
     const taskctl = YAML.parse(tree.read('taskctl.yaml', 'utf8'));
-    if (taskctl.pipelines) {
-        if (visualRegression) {
-            // Add local tasks
-            taskctl.pipelines.dev.push({ task: 'e2e:local' });
-            taskctl.pipelines.fe.push({ task: 'e2e:local' });
-            // Add ci tasks
-            taskctl.pipelines.nonprod.push({ task: 'e2e:ci' });
-            taskctl.pipelines.prod.push({ task: 'e2e:ci' });
-        } else {
-            // Add tasks
-            taskctl.pipelines.dev.push({ task: 'e2e' });
-            taskctl.pipelines.fe.push({ task: 'e2e' });
-            taskctl.pipelines.nonprod.push({ task: 'e2e' });
-            taskctl.pipelines.prod.push({ task: 'e2e' });
-        }
+    if (taskctl.pipelines && visualRegression) {
+        taskctl.pipelines.updatesnapshots = [{ task: 'e2e:updatesnapshots' }];
```

Comment:
Also confirming here, but is this only added if visual regression == native?

Reply:
it's only called when native